sympy/utilities/lambdify.py

```diff
@@ -233,6 +286,19 @@ def lambdify(args, expr, modules=None, printer=None, use_imps=True,
         >>> f(array([1, 2, 3]), array([2, 3, 5]))
         [-2.18503986 -0.29100619 -0.8559934 ]
 
+        Use of scipy module:
+
+        >>> import scipy
+        >>> import numpy
+        >>> a = numpy.arange(10)
+        >>> a
+        [0 1 2 3 4 5 6 7 8 9]
+        >>> expr=sin(x)
```

Comment:
Please check your code with a linter that checks for PEP8 issues, e.g. spaces around the `=` here.